node/tests/TestUtilities.ts

```diff
@@ -610,6 +611,17 @@ export async function transactionTest(
     responseData.push(["dbsize()", 0]);
     baseTransaction.set(key1, "bar");
     responseData.push(['set(key1, "bar")', "OK"]);
+
```

Comment:
Don't we have set with options in transaction IT? Add if no, please.

Reply:
We indeed do not, added, should the title of the PR be updated to reflect the `Set` changes?